src/main/java/ua/lviv/iot/parkingServer/natscontroller/NatsParkingGetByIdController.kt

```diff
@@ -0,0 +1,33 @@
+package ua.lviv.iot.parkingServer.natscontroller
+
+import com.example.ParkingOuterClass.GetByIdParkingRequest
+import com.example.ParkingOuterClass.GetByIdParkingResponse
+import com.google.protobuf.Parser
+import io.nats.client.Connection
+import org.springframework.stereotype.Component
+import ua.lviv.iot.parkingServer.converter.ParkingConverter
+import ua.lviv.iot.parkingServer.service.interfaces.ParkingServiceInterface
+
+@Component
+class NatsParkingGetByIdController(
+    private val converter: ParkingConverter,
+    private val service: ParkingServiceInterface,
+    override val connection: Connection,
+) : NatsController<GetByIdParkingRequest, GetByIdParkingResponse> {
```

Comment:
```suggestion
): NatsController<GetByIdParkingRequest, GetByIdParkingResponse> {
```

Reply:
fix this in all classes

Reply:
please, don't touch. Everything is ok here